### PR TITLE
GH-1122 Migrating routing e2e tests to cypress

### LINKED ISSUE #1122: migrate smoke/routing to cypress
See #1102 for original spike.
Take a look at big happy path for example.
See cypress video in our wiki.
Likely doesn't have login but if it needs login, set this to FR2 (and do after #1216).

### PR DESCRIPTION
**Pull request by hackergil**
Closes #1122

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const PRODUCT_ID = '1992693';
+const PRODUCT_NAME = 'DSC-T90';
+
+context('Configurable routing', () => {
+  it('should show product page by product ID', () => {
+    cy.visit(`/product/${PRODUCT_ID}`);
+    const productSummary = cy.get('cx-product-summary');
+    productSummary.should('exist');
```
Comment (marlass) [inline projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts]: The same thing here. This exist assertion doesn't bring value.

File: projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    productSummary.within(() => {
+      cy.get('.name').should('contain', PRODUCT_NAME);
+    });
+
+    cy.visit(`/product/${PRODUCT_NAME}/${PRODUCT_ID}`);
+    expect(true).to.equal(true);
```
Comment (marlass) [inline projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts]: And that is strange :)

File: projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts (hunk 1 above)
Comment (hackergil) [reply projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts]: Removed

File: projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts (hunk 2 above)
Comment (hackergil) [reply projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts]: Forgot to delete that. Done now.

**Pull request merged**